srcpkgs/python3-jellyfish/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-jellyfish'
+pkgname=python3-jellyfish
+version=0.7.1
+revision=1
+wrksrc="jellyfish-${version}"
+build_style=python3-module
+pycompile_module="jellyfish"
+hostmakedepends=" python3-setuptools"
+makedepends="python3-devel"
+short_desc="Python3 library for approximate and phonetic matching of strings"
+maintainer="Duncaen <duncaen@voidlinux.org>"
```

Comment:
@Duncaen Do you have interesting in this package ?

Reply:
Added it for beets, which I use, so kinda.